src/components/DeckMechanisms/index.js

```diff
@@ -432,6 +446,7 @@ export default class DeckMechanisms extends React.Component {
       turn: this.state.turn,
       RNG: this.state.RNG,
       hasCycledThisTurn: this.state.hasCycledThisTurn,
+      noUnitsOnFirstTurn: this.noUnitsOnFirstTurn,
```

Comment:
I'm not sure if this is needed, but since all the other properties were returned I added it as well